.slugignore

```diff
@@ -0,0 +1 @@
+spec/
```

Comment:
I can think of  `scripts/` but unsure how big of a difference it would make

Reply:
```shell
devto git:(master) du -chs spec scripts
8.6M	spec
8.0K	scripts
8.6M	total
```

:D

Reply:
Nothing here is huge :-/ Tbh the whole thing discussed in the issue probably gets resolved by running `git gc` on the slug.